tRecorderApi/api/views/lang_list.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import views, status
+from rest_framework.parsers import JSONParser
+import json
+from api.models import Take, Language
+from rest_framework.response import Response
+from operator import itemgetter
+
+class getLangsView(views.APIView):
+    parser_classes = (JSONParser,)
+
+    def post(self, request):
+        data = json.loads(request.body)
+        allTakes = Take.objects.all().values()
+        allTakes = allTakes.filter(is_source = False)
```

Comment:
@lcheng49 Can we init and assign the variable in one line? Does it need to be 2?